sos/report/plugins/__init__.py

```diff
@@ -585,7 +585,18 @@ def __init__(self, commons):
         self.set_predicate(SoSPredicate(self))
 
     def get_default_plugin_opts(self):
+        # The default of options that have a global argument counterpart with
+        # the same name must use the same default in both places.
         return {
+            'maxhours': PluginOpt(
+                'maxhours', default=None, val_type=int,
+                desc='Escapes archived files older (according to `mktime`) '
+                'than this many hours'
+            ),
+            'all_logs': PluginOpt(
```

Comment:
I'm new to this, but I assumed looking at the [kernel plugin `with-timer` definition](https://github.com/sosreport/sos/blob/e48a41dd6da6178cc04ce6a6d7685814195e4384/sos/report/plugins/kernel.py#L39) and how it is [accessed as `with-timer` and not `with_timer`](https://github.com/sosreport/sos/blob/e48a41dd6da6178cc04ce6a6d7685814195e4384/sos/report/plugins/kernel.py#L144) that there is no translation on the `PluginOpt`.

I looked a bit into the `get_option` and `PluginOpt` and didn't find any translation mechanism so I went with the solution that required fewer code changes.

But it is true that it would be weird to call it `--all-logs` only to have to use `all_logs` in the plugins.

I'll see if I can use `all-logs` everywhere.

Reply:
Done